packages/picasso.js/src/core/chart/index.js

```diff
@@ -176,6 +177,54 @@ import componentCollectionFn from './component-collection';
  * }
  */
 
+/**
+ * @typedef {object} RendererSettings
+ * @property {RendererSettings~TransformFunction} [transform] Setting for applying transform without re-rendering the whole component completely.
+ * @property {RendererSettings~CanvasBufferSize} [canvasBufferSize] Specifies the size of buffer canvas (used together with transform setting).
+ * @experimental
+ */
+
+/**
+ * Should return a transform object if transformation should be applied, otherwise undefined or a falsy value.
+ * Transforms can be applied with the canvas, svg and dom renderer.
+ * !Transform is applied when running chart.update with partialData set to true, see example.
```

Comment:
it is also applied when partialData is false or missing

Reply:
Oh, you are right about that. Good catch! I haven't tested that scenario... I will do that and see if it makes sense/works in that case...

Reply:
I tested it with running just `chart.update()` as well, and that works as well. It might not be the best way to do it, but that's another story. I adjusted the description accordingly anyway!